routes/index.js

```diff
@@ -27,8 +28,10 @@ router.get('/', function(req, res, next) {
   /**
    * Load the entries file
```

Comment:
Ths comment is no longer accurate since you're not loading a file, but reading from a database.